score-lang-api/src/main/java/org/openscore/lang/api/Slang.java

```diff
@@ -26,25 +26,16 @@
 public interface Slang {
 
     /**
-     * Compile a flow written in slang
+     * Compile a flow/operation written in slang
```

Comment:
as @SamMarkowitz commented on another API it's preferable to use 'or' and not a slash